src/components/live-traffic.vue

```diff
@@ -95,11 +105,13 @@ export default class extends DashboardWidget {
     this.BoxedCountFlight = Object.keys(flights).length;
     this.BoxedCountAirline = airlines.size;
     if (this.BoxedCountAirline !== 0) {
-      this.BoxedMaxSpeed = Math.max.apply(
-        0,
-        flights.map((f: Flight) => f.speed),
-      );
+      const maxSpeedFlight = flights.reduce((max: Flight, f: Flight) => {
+        return ( f.speed > max.speed ? f : max );
```

Comment:
fails il flights is empty?
also, you don't need return here (nor the braces)

Reply:
the if condition should fail if flights is empty, thanks for return (and braces) 😬